tests/test_command.py

```diff
@@ -198,6 +198,25 @@ def test_history_indicate_current(self):
         )
 
 
+class MutationTest(TestBase):
+    __only_on__ = "sqlite"
+
+    @classmethod
+    def setup_class(cls):
+        cls.bind = _sqlite_file_db()
+        cls.env = staging_env()
+        cls.cfg = _sqlite_testing_config()
+
+    @classmethod
+    def teardown_class(cls):
+        clear_staging_env()
+
+    def test_current(self):
+        command.current(self.cfg)
+        engine = self.bind
+        assert not engine.dialect.has_table(engine, "alembic_version")
```

Comment:
This currently fails the tests since has_table expects a connection.
You should update with `with engine.connect() as conn: ` and pass `conn` to `has_table`

also probably instead of `assert` you can use `is_false` imported from `alembic.testing`

Reply:
Interesting, they pass locally when running `tox`. Can you link to a CI output where you've found that exception?

Reply:
that is because sqlalchemy master has that requirements

Reply:
note that we could probably also use `_connectable_has_table` that does the right thing

Reply:
oh sorry, the ci link is this
https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/1970
https://jenkins.sqlalchemy.org/job/alembic_gerrit/pyv=py37,sqla=sqlamaster/487/testReport/

Reply:
>This currently fails the tests since has_table expects a connection.

Thanks for the hint, fixed in ec486e9.

>also probably instead of assert you can use is_false imported from alembic.testing

See 9665c48.

Reply:
>note that we could probably also use _connectable_has_table that does the right thing

See also ec486e9.